account_move_line_menu/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Menu option available at Accounting > Accounting > Miscellaneous > Journal items.
```

Comment:
non-blocking comment. Enterprise edition has `Accounting` menu and community edition has `Invoicing` menu.

Reply:
Good point, better add both cases.